hummingbot/strategy/pure_market_making/pure_market_making.pxd

```diff
@@ -17,17 +17,18 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         double _bid_place_threshold
         double _ask_place_threshold
         double _cancel_order_wait_time
-        double _time_to_cancel
         double _order_size
         double _status_report_interval
         double _last_timestamp
         double _limit_order_min_expiration
+        list _radar_relay_type_exchanges
```

Comment:
... which means this should be a `set` rather than a list.

Reply:
changed